runtime/ejs-runloop-darwin.m

```diff
@@ -44,6 +54,48 @@ -(void)runTask
 
 @end
 
+@implementation DelayedTaskObj
++(void)runDelayedTask:(NSTimer*)timer
+{
+    NSDictionary *taskData = timer.userInfo;
```

Comment:
is there any reason why this couldn't be a DelayedTaskObj instance?  and instead of the objectForKey calls, we could just have `.` or accessors calls?